release-notes/opensearch.release-notes-1.3.7.md

```diff
@@ -0,0 +1,23 @@
+## 2022-11-30 Version 1.3.6 Release Notes
+
+### Upgrades
+* Updated Jackson Databind to 2.13.4.2 ([#4785](https://github.com/opensearch-project/OpenSearch/pull/4785))
+* Upgrade netty to 4.1.84.Final ([#4919](https://github.com/opensearch-project/OpenSearch/pull/4919))
+* OpenJDK Update (October 2022 Patch releases) ([#5016](https://github.com/opensearch-project/OpenSearch/pull/5016))
+* Upgrade com.netflix.nebula:nebula-publishing-plugin to 4.6.0 and gradle-docker-compose-plugin to 0.14.12 ([#5316](https://github.com/opensearch-project/OpenSearch/pull/5136))
+* Updated Jackson to 2.14.0 ([#5113](https://github.com/opensearch-project/OpenSearch/pull/5113))
+* Updated Jackson to 2.14.1 ([#5356](https://github.com/opensearch-project/OpenSearch/pull/5356))
+
+
+### Enhancements
+* copy build.sh over from opensearch-build ([#4992](https://github.com/opensearch-project/OpenSearch/pull/4992))
```

Comment:
I think we can apply the same criteria here as is [documented for the changelog](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#changelog). All these test and build-related changes can be omitted. I think that will only leave the dependency upgrades above and PR 5149 below.